sktime/pipeline/_make_pipeline.py

```diff
@@ -20,25 +20,20 @@ def make_pipeline(*steps):
 
     Examples
     --------
-    >>> from sktime.datasets import load_airline
-    >>> y = load_airline()
-
     Example 1: forecaster pipeline
 
-    >>> from sktime.datasets import load_airline
-    >>> from sktime.forecasting.trend import PolynomialTrendForecaster
     >>> from sktime.pipeline import make_pipeline
     >>> from sktime.transformations.series.exponent import ExponentTransformer
-    >>> y = load_airline()
+    >>> from sktime.forecasting.trend import PolynomialTrendForecaster
     >>> pipe = make_pipeline(ExponentTransformer(), PolynomialTrendForecaster())
     >>> type(pipe).__name__
     'TransformedTargetForecaster'
 
     Example 2: classifier pipeline
 
-    >>> from sktime.classification.distance_based import KNeighborsTimeSeriesClassifier
     >>> from sktime.pipeline import make_pipeline
     >>> from sktime.transformations.series.exponent import ExponentTransformer
+    >>> from sktime.classification.distance_based import KNeighborsTimeSeriesClassifier
```

Comment:
I'm not sure what effect does it have. At least by `isort`, previous seemed better style. Does it affect the hosted docs?

Reply:
It was done so that it is consistent across all other in the docstring examples. following this steps -> import make_pipeline, then import exponenttransformer then import estimators.